webpack/components/extensions/HostDetails/Tabs/TracesTab.js

```diff
@@ -6,14 +6,13 @@ import { useSelector, useDispatch } from 'react-redux';
 import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
 import EnableTracerEmptyState from './EnableTracerEmptyState';
 import TableWrapper from '../../../Table/TableWrapper';
-import useSet from '../../../Table/TableHooks';
+import { useSelectionSet } from '../../../Table/TableHooks';
```

Comment:
How difficult would it be to add the concern to the host traces controller and go all out with `useBulkSelect` here?

Reply:
I can try adding here. Shouldn't be hard. But isn't this PR large already? 
That being said I can add it to this PR if you prefer.

Reply:
I would only want it if it's super easy!  Let's punt this to the next PR for now